src/systems/userinput/userinput.js

```diff
@@ -274,8 +279,19 @@ AFRAME.registerSystem("userinput", {
           const activeDevice = this.activeDevices.items[i];
           const mapping = vrGamepadMappings.get(activeDevice.constructor);
           mapping && this.registeredMappings.add(mapping);
-          if (activeDevice instanceof ViveControllerDevice && activeDevice.isViveWand) {
-            this.registeredMappings.add(viveWandUserBindings);
+
+          if (activeDevice instanceof ViveControllerDevice && activeDevice.gamepad) {
+            console.log(activeDevice);
+            if (activeDevice.gamepad.id === "HTC Vive Focus Plus Controller") {
+              //HTC Vive Focus Plus Controller
+              this.registeredMappings.add(viveFocusPlusUserBindings);
+            } else if (activeDevice.gamepad.axes.length === 4) {
```

Comment:
Do wands and index share a string id and should we match on it here? (Or just default all other controllers to one of these since it's out best guess?)